editor/export/editor_export_platform.cpp

```diff
@@ -881,7 +881,11 @@ Vector<String> EditorExportPlatform::get_forced_export_files() {
 		files.push_back(icon);
 	}
 	if (!splash.is_empty() && FileAccess::exists(splash) && icon != splash) {
-		files.push_back(splash);
+		if (!splash.ends_with(".png")) {
+			ERR_PRINT("Invalid boot splash : " + splash + ". The only supported format is PNG. Skipping export.");
+		} else {
+			files.push_back(splash);
+		}
```

Comment:
I think the better error message in `setup_boot_logo()` is enough to solve the problem.

`get_forced_export_files()` does not seem to be a right place for this kind of checks.

Reply:
I was thinking about how we can prevent user doing this at the first place, I agree the export is not a right place lol.